src/core/SkillRequest.ts

```diff
@@ -87,6 +109,23 @@ export class SkillRequest {
         this._json.request.token = token;
         return this;
     }
+
+    public inSkillPurchaseResponse(requestName: string, 
+        productId: string, 
```

Comment:
The implementation doesn't match the documentation. I prefer the productId as the 3rd param as shown in the docs.

Reply:
Hi Mark, which documentation are you referring to? And FYI, I added this based on your code and this blog post:
https://developer.amazon.com/blogs/alexa/post/53c2319a-4cc9-4067-8274-1472a0463d6e/understanding-the-in-skill-purchasing-experience

Reply:
You own README.md changed in the PR has the productId as the 3rd param. I agree that it should be 3rd. The code in the PR has it as 2nd param.